addon/services/nav-stacks.js

```diff
@@ -34,12 +39,38 @@ export default class NavStacks extends Service {
 
   notifyTransitionEnd() {
     this._runningTransitions--;
+    next(() => {
+      this._maybeResolveIdle();
+    });
   }
 
   runningTransitions() {
     return this._runningTransitions;
   }
 
+  waitUntilTransitionIdle() {
```

Comment:
Where is this used?

Reply:
It's used in the AnimationAware mixin in yapp-core.